test/e2e_node/image_list.go

```diff
@@ -84,16 +85,12 @@ func updateImageAllowList(ctx context.Context) {
 	} else {
 		e2epod.ImagePrePullList.Insert(gpuDevicePluginImage)
 	}
+	// TODO(1.28+): unify the sample device plugin image, use the latest for everything
```

Comment:
version reference in TODO is confusing. Do you want to do it in 1.28 in separate PR?